documentation/guide-logging.asciidoc

```diff
@@ -111,4 +111,34 @@ In order to correlate separate HTTP requests to services belonging to the same u
 == Monitoring
 In highly distributed systems (from clustering up to microservices) it might get tedious to search for problems and details in log files. Therefore, we recommend to setup a central log and analysis server for your application landscape. Then you feed the logs from all your applications (using http://logstash.net/[logstash]) into that central server that adds them to a search index to allow fast searches (using http://www.elasticsearch.org/[elasticsearch]). This should be completed with a UI that allows dashboards and reports based on data aggregated from all the logs.
 This is addressed by https://www.elastic.co/webinars/introduction-elk-stack[ELK] or https://www.graylog.org/[Graylog].
- 
\ No newline at end of file
+
+=== Adding custom values to JSON log
+
+When you use a external system for gathering distrubited logs, we strongly suggest that you use a JSON based log format (e.g. by using the provided `logback.xml`, see above).
```

Comment:
Do you know why you mix features in your PRs several times? Something is odd with github? Or the way you manage your feature-brances? Or the way you merge?

Reply:
It is not a general problem there were conflicts because I started with this task here before the PR for the batch topic itself was merged. Sadly I put the minor change for logback on by develop branch (instead of creating a feature branch). That mixed everything up. I am ashame...

I removed that commit from the PR.